src/vendor_upstream/core/globalThisPolyfill.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule globalThisPolyfill
+ */
+
+const globalThisPolyfill =
```

Comment:
nit: can we simply write it like this` const globalThisPolyFill = globalThis || window || global || self || {} `

Reply:
Unfortunately that won't work if any of these are not defined.